src/main/java/com/vinylteam/vinyl/dao/jdbc/JdbcUserDao.java

```diff
@@ -1,6 +1,95 @@
 package com.vinylteam.vinyl.dao.jdbc;
 
+import com.vinylteam.vinyl.dao.DBDataSource;
 import com.vinylteam.vinyl.dao.UserDao;
+import com.vinylteam.vinyl.dao.jdbc.mapper.UserRowMapper;
+import com.vinylteam.vinyl.entity.User;
+import org.postgresql.util.PSQLException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.*;
+import java.util.Optional;
 
 public class JdbcUserDao implements UserDao {
+
+    private final String COUNT_ALL = "SELECT COUNT(*) FROM public.users";
+    private final String FIND_BY_EMAIL = "SELECT * FROM public.users" +
+            " WHERE email=?";
```

Comment:
What I said about using stars in query? Fix it please. You should specify all columns.